EIPS/eip_on-chain_NFT_royalty.md

```diff
@@ -0,0 +1,90 @@
+---
```

Comment:
```suggestion
---
eip: 7526
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.